travelwise/frontend/templates/travel/create.html

```diff
@@ -42,13 +35,52 @@ <h2>Activities</h2>
     </div>
   </div>
 </main>
-
+{{activities|json_script:"activitiesJson"}}
```

Comment:
Interesting way of doing this...

Reply:
The data needs to be machine-readable on the frontend in order for sorting to work. I don't think there's a much better way to do this.